Documentation/event-reference.md

```diff
@@ -89,11 +89,24 @@ If the required `xdm` key is not present in the event data payload, the event is
 | --- | ---------- | -------- | ----------- |
 | xdm | <code>[String:&nbsp;Any]</code> | Yes | XDM formatted data; use an `XDMSchema` implementation for better XDM data ingestion and data format control. |
 | data | <code>[String:&nbsp;Any]</code> | No | Optional free-form data associated with this event. |
-| datastreamIdOverride | `String` | No | Optional Datastream identifier used to override the default datastream identifier set in the Edge configuration. |
-| datastreamConfigOverride | <code>[String:&nbsp;Any]</code> | No | Optional Datastream configuration used to override individual settings from the default datastream configuration. |
+| config | <code>[String:&nbsp;Any]</code> | No | Optional config overrides. |
 | datasetId | `String` | No | Optional custom dataset ID. If not set, the event uses the default Experience dataset ID set in the datastream configuration. |
 | request | <code>[String:&nbsp;Any]</code> | No | Optional request parameters. |
 
+config
+
+| Key | Value type | Required | Description |
+| --- | ---------- | -------- | ----------- |
+| config.datastreamIdOverride | `String` | No | Optional Datastream identifier used to override the default datastream identifier set in the Edge configuration. |
+| config.datastreamConfigOverride | <code>[String:&nbsp;Any]</code> | No | Optional Datastream configuration used to override individual settings from the default datastream configuration. |
```

Comment:
- We should probably list just the inner key "datastreamIdOverride" / "datastreamConfigOverride" here, so users don't get confused that the key name is prefixed with "config."
- Same for request table